sdk/spring-sdk/src/test/scala/kalix/springsdk/impl/ViewDescriptorFactorySpec.scala

```diff
@@ -20,23 +20,26 @@ import com.google.protobuf.Descriptors.FieldDescriptor.JavaType
 import kalix.JwtMethodOptions.JwtMethodMode
 import kalix.springsdk.impl.reflection.ServiceIntrospectionException
 import kalix.springsdk.testmodels.subscriptions.PubSubTestModels.SubscribeOnTypeToEventSourcedEventsWithMethodWithState
-import kalix.springsdk.testmodels.view.ViewTestModels.IllDefineUserByEmailWithStreamUpdates
-import kalix.springsdk.testmodels.view.ViewTestModels.SubscribeToEventSourcedEvents
-import kalix.springsdk.testmodels.view.ViewTestModels.SubscribeToEventSourcedEventsWithMethodWithState
-import kalix.springsdk.testmodels.view.ViewTestModels.TransformedUserView
-import kalix.springsdk.testmodels.view.ViewTestModels.TransformedUserViewUsingState
-import kalix.springsdk.testmodels.view.ViewTestModels.TransformedUserViewWithJWT
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByEmailWithGet
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByEmailWithPost
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByEmailWithPostRequestBodyOnly
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByEmailWithStreamUpdates
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByNameEmailWithPost
-import kalix.springsdk.testmodels.view.ViewTestModels.UserByNameStreamed
-import kalix.springsdk.testmodels.view.ViewTestModels.ViewWithMethodLevelAcl
-import kalix.springsdk.testmodels.view.ViewTestModels.ViewWithNoQuery
-import kalix.springsdk.testmodels.view.ViewTestModels.ViewWithServiceLevelAcl
-import kalix.springsdk.testmodels.view.ViewTestModels.ViewWithSubscriptionsInMixedLevels
-import kalix.springsdk.testmodels.view.ViewTestModels.ViewWithTwoQueries
+import kalix.springsdk.testmodels.view.ViewTestModels.{
```

Comment:
not sure about the convention, but it looks like this will overridden in other PRs, in my case it's flat: https://github.com/lightbend/kalix-jvm-sdk/pull/1236/files#diff-51a4487b6b60e90723ded89d0ecafd7a6296804c23d3015ce2bd691080191b04, probably some intellij formatting setting

Reply:
Thanks. yeah, I'll see if I find the setting and disable but I also think we should probably add this to scalafmt rules if we want to enforce the convention.